yarn.lock

```diff
@@ -1393,7 +1393,7 @@
     ripemd160 "^2.0.2"
     sha3 "^2.1.3"
 
-"@eslint/eslintrc@^0.4.0", "@eslint/eslintrc@^0.4.1":
+"@eslint/eslintrc@^0.4.1":
```

Comment:
Could you revert all the yarn.lock changes?
We should commit them only when package.json is changed or in a separate dependency upgrade PR.